pandas/tests/arithmetic/test_numeric.py

```diff
@@ -72,6 +73,25 @@ def test_compare_invalid(self):
         tm.assert_series_equal(a / b, 1 / (b / a))
 
 
+class TestNumericArraylikeArithmeticWithBool:
+    @pytest.mark.parametrize("num", [1.0, 1])
+    def test_array_like_bool_and_num_op_coerce(
+        self, num, all_arithmetic_functions, box_with_array
+    ):
+        # GH 18549
+        op = all_arithmetic_functions
+        expected = [op(num, num)]
+        expected = tm.box_expected(expected, box_with_array)
+        bool_box = tm.box_expected([True], box_with_array)
+        try:
```

Comment:
rather than catching you should return on the operators that we don't want to test, making this more explicit